input/websocket.js

```diff
@@ -9,18 +9,14 @@ function create ({address, parse}) {
       socket.on('open', _ => {
         pipeline.status(null, 'Listening to ' + address)
       })
-      socket.on('message', stringMessage => {
-        const json = JSON.parse(stringMessage)
-        const messages = Array.isArray(json) ? json : [json]
-        messages.forEach(message => {
-          let log
-          try {
-            log = parse(message)
-          } catch (err) {
-            return pipeline.error(err)
-          }
-          pipeline.success(log)
-        })
+      socket.on('message', message => {
+        let log
+        try {
+          log = parse(JSON.parse(message))
+        } catch (err) {
+          return pipeline.error(err)
+        }
+        pipeline.success(log)
```

Comment:
I understand that this code is copied from other inputs, but would it make sense to write it like that and change  the style everywhere else?

```
try {
  const log = parse(JSON.parse(message))
  pipeline.success(log)
} catch (err) {
  pipeline.error(err)
}
```
 ```log``` is a constant used only in the block, and there is no purpose returning the pipeline.error. what do you think @benfle ?

Reply:
Yup should be the same